src/components/calcite-flow/calcite-flow.scss

```diff
@@ -1,28 +1,27 @@
-$start-alpha: 0.5;
-
 :host {
+  //FIXME: Should these extend be removed?
```

Comment:
looks like `%component-host` is pretty opinionated. It sets a bg color, color, font-size, box-sizing, and box sizing for _all children_. We could convert all of those other than the nested child wildcard to tailwind. 

We could keep the child selection parts of `%component-host` so that it becomes something like:

```
%component-host {
  box-sizing: border-box;
  border: 0 solid;
  * {
      box-sizing: border-box;
      border: 0 solid;
  }
}
```

which would also solve our border woes and allow the pattern to be generic enough that it works for all our components (less opinionated). For now I would leave it but I'd like to hear @asangma 's thoughts on this.

Reply:
I can leave it for now. I'll also leave the comment there for when we stumble on it again we can fix it if needed?

Reply:
I talked with Alan, we can remove %component-spacing everywhere after tailwind is added within the file. %component-host will be taken care of later by Alan.